otter/test/models/test_cass_models.py

```diff
@@ -2527,6 +2557,68 @@ def test_add_scaling_policy_cron(self):
         self.assertEqual(result, [pol])
 
 
+class ScalingGroupWebhookMigrateTests(SynchronousTestCase):
+    """
+    Tests for webhook migration functions in :obj:`CassScalingGroup`
+    """
+
+    def setUp(self):
+        """
+        Sample CassScalingGroup object
+        """
+        self.store = CassScalingGroupCollection(None, None)
+
+    def test_webhook_index_only(self):
+        """
+        Test `get_webhook_index_only`
```

Comment:
I prefer to avoid that; the test is always testing something more nuanced than the public documentation.  In this case, one obvious distinction is that `get_webhook_index_only`'s docstring does not note that the effects take place in parallel, yet this test cares and will fail if the effect's intent is sequential rather than parallel.